docs/src/reference/gremlin-variants.asciidoc

```diff
@@ -797,7 +797,7 @@ The following table describes the various configuration options for the Gremlin
 |connectionPool.keyStore |The private key in JKS or PKCS#12 format. |_none_
 |connectionPool.keyStorePassword |The password of the `keyStore` if it is password-protected. |_none_
 |connectionPool.keyStoreType |`JKS` (Java 8 default) or `PKCS12` (Java 9+ default)|_none_
-|connectionPool.maxContentLength |The maximum length in bytes that a message can be sent to the server. This number can be no greater than the setting of the same name in the server configuration. |65536
+|connectionPool.maxResponseContentLength |The maximum length in bytes that a message can be received from the server. |`Integer.MAX_VALUE`
```

Comment:
Might be nice to just write out 2147483647 instead of Integer.MAX_VALUE. Saves people a step from having to look it up.

Reply:
Changed to 2147483647